Lombiq.Hosting.MediaTheme.Targets/Readme.md

```diff
@@ -0,0 +1,37 @@
+# Lombiq Hosting - MediaTheme MSBuild Targets
+
+[![Lombiq.Hosting.MediaTheme.Targets NuGet](https://img.shields.io/nuget/v/Lombiq.Hosting.MediaTheme.Targets?label=Lombiq.Hosting.MediaTheme.Targets)](https://www.nuget.org/packages/Lombiq.Hosting.MediaTheme.Targets/)<!-- #spell-check-ignore-line -->
```

Comment:
Why do have to ignore this line?

Reply:
The error is not from the spell checker but the Markdown linter in Node.js Extensions 😉

Reply:
Okay, but what's the problem with this line?

Reply:
I checked and I honestly don't know.

Reply:
Now I added <!-- markdownlint-disable --> but that didn't help either... Also I couldn't find this "_doubled-spaces: Found doubled spaces._" error anywhere documented in markdownlint. :O

Reply:
Oh it is actually textlint